example/classifier/odp_classifier.c

```diff
@@ -100,6 +107,27 @@ static int parse_args(int argc, char *argv[], appl_args_t *appl_args);
 static void print_info(char *progname, appl_args_t *appl_args);
 static void usage(void);
 
+static inline int check_ci_pass_count(appl_args_t *args)
+{
+	int i, j;
+	uint64_t count;
+
+	if (args->ci_pass_count == 0)
+		return 0;
+
+	for (i = 0; i < args->ci_pass_count; i++) {
+		for (j = 0; j < args->policy_count; j++) {
+			if (strcmp(args->stats[j].cos_name,
+				   args->cpass_count[i].cos_name)) {
+				count = odp_atomic_load_u64(&args->stats[i].queue_pkt_count);
+				if (args->cpass_count[i].count > count)
+					return -1;
```

Comment:
To make debugging easier you could also print the cos name here.

Reply:
Ack